web/sample.env

```diff
@@ -0,0 +1,5 @@
+# Make your API Key from https://console.cloud.google.com/apis/credentials
+# Reuse the OAuth Client ID used for server/.env
+
+VITE_CLIENT_ID=
```

Comment:
These are not Vite keys, it's for Google's OAuth (as also noted by the comment)

Reply:
I have renamed the variables for clarity.
Also take a look here - https://vite.dev/guide/env-and-mode
> Only VITE_SOME_KEY will be exposed as import.meta.env.VITE_SOME_KEY to your client source code, but DB_PASSWORD will not.